pages/glossary.md

```diff
@@ -0,0 +1,157 @@
+
+## Externally-Owned Account
+
+An externally-owned account (also known as EOA) is one of the two types of Ethereum accounts. It's controlled by a private key, it has no code and users can send messages by creating and signing Ethereum transactions.
+
+See also:
+- [Ethereum Accounts](https://ethereum.org/en/developers/docs/accounts) on ethereum.org
+- [Ethereum Whitepaper](https://ethereum.org/en/whitepaper/#ethereum-accounts) on ethereum.org
+
+## Gasless Transaction
+
+Gasless transactions (also known as meta-transactions) are Ethereum transactions that are executed by a third party called [relayer](#relayer) on behalf of a [smart account](#smart-account) to abstract the use of gas. Users need to sign a message (instead of the transaction itself) with information about the transaction they want to execute and a relayer will be in charge of creating the Ethereum transaction, signing it and executing it, paying for the gas costs. The main benefit is that users can interact with the blockchain without needing to hold the native token in their account.
+
+See also:
+- [Relay Kit documentation](./sdk-relay-kit) on docs.safe.global
+
+## Network
+
+A blockchain network is a collection of interconnected computers that utilize a blockchain protocol for communication.
+Decentralized networks allow users to send transactions, that are processed on a distributed ledger with a consensus mechanism ensuring the batching,
+verification, and acceptance of data into blocks. This structure enables the development of applications without the need for a central authority or server.
+
+See also:
+- [Networks](https://ethereum.org/en/developers/docs/networks) on ethereum.org
+
+## Owner
+
+A Safe owner is one of the accounts that control a given Safe, being the only ones that can manage the configuration of a Safe and approve transactions. They can be either [externally-owned accounts](#externally-owned-account) or [smart accounts](#safe-smart-accounts). The [threshold](#threshold) of a Safe defines how many owners need to approve a Safe transaction to make it executable.
+
+See also:
+- [OwnerManager.sol](https://github.com/safe-global/safe-smart-account/blob/main/contracts/base/OwnerManager.sol) on github.com
+
+## Relayer
+
+A relayer is a third-party service serving as an intermediary between users' accounts and [blockchain networks](#network). It executes transactions on behalf of users and covers the associated execution costs, which may or may not be claimed.
+
+See also:
+- [What's Relaying?](https://docs.gelato.network/developer-services/relay/what-is-relaying) on docs.gelato.network
+
+## Safe{DAO}
+
+The Safe{DAO} is the [Decentralized Autonomous Organization](https://ethereum.org/dao) (DAO) that aims to foster a vibrant ecosystem of applications and wallets leveraging Safe accounts. This will be achieved through data-backed discussions, grants, ecosystem investments, as well as providing developer tools and infrastructure.
```

Comment:
```suggestion
The Safe{DAO} is the [Decentralized Autonomous Organization](https://ethereum.org/dao) (DAO) that aims to foster a vibrant ecosystem of applications and wallets leveraging Safe accounts. This will be achieved through data-backed discussions, grants, and ecosystem investments, as well as providing developer tools and infrastructure.
```